baselines/flwr_baselines/publications/wav2vec2/tedlium/docs/configs/w2v2.yaml

```
@@ -0,0 +1,180 @@
+# ################################
+# Model: seq2seq ASR on TIMIT with wav2vec2 + CTC/Attention
+# Authors:
+# * Titouan Parcollet 2021
+# ################################
+
+# Seed needs to be set at top of yaml, before objects with parameters are made
+seed: 1234
+__set_seed: !!python/object/apply:torch.manual_seed [!ref <seed>]
+output_folder: !ref docs/results/fl_wav2vec2/<seed>
+wer_file: !ref <output_folder>/wer.txt
+save_folder: !ref <output_folder>/save
+train_log: !ref <output_folder>/train_log.txt
+
```

```suggestion
label_encoder: material/label_encoder.txt
```
Avoid the possibility of missing error.